<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CvMobMainWindow</class>
 <widget class="QMainWindow" name="CvMobMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CvMob</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="centralLayout"/>
  </widget>
  <widget class="QDockWidget" name="openVideos">
   <property name="windowTitle">
    <string>Opened videos</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="openButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="openedVideosList"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Data tables</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="dataWidget">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1677215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="trajectoriesTab">
        <attribute name="title">
         <string>Trajectories</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QTreeView" name="trajectoriesView">
           <property name="toolTip">
            <string extracomment="Trajectory data"/>
           </property>
           <property name="whatsThis">
            <string extracomment="Trajectory data2"/>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="distancesTab">
        <attribute name="title">
         <string>Distances</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTreeView" name="distancesView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="anglesTab">
        <attribute name="title">
         <string>Angles</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTreeView" name="anglesView">
           <property name="toolTip">
            <string extracomment="Trajectory data"/>
           </property>
           <property name="whatsThis">
            <string extracomment="Trajectory data2"/>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="graphsDockWidget">
   <property name="windowTitle">
    <string>Graphs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="graphsDockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="graphsGroup">
       <property name="title">
        <string>Enabled graphs</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="xGraphCheckBox">
          <property name="text">
           <string>X (Trajectory)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="yGraphCheckBox">
          <property name="text">
           <string>Y (Trajectory)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="speedCheckBox">
          <property name="text">
           <string>Speed (Trajectory)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="accelerationCheckBox">
          <property name="text">
           <string>Acceleration (Trajectory)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="angleCheckBox">
          <property name="text">
           <string>Angle</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open a new video</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
